### PR TITLE
refactor player.hasStarted()

### PR DESCRIPTION
**Pull request by kocoten1992**
update doc on hasStarted()

I feel 'request' are more suitable than hasStarted (as a variable) because when I do "if (request === undefined)", it mean if you don't have any request, then we just return current status, it a bit more clear

## Requirements Checklist
- [x] Docs/guides updated

### CODE REVIEW

**Comment by kocoten1992** [comment]
Hi @gkatsev , does this style of writing clearer than the previous one.

Cause I'm running through the source and see a lot of js write at old style, if this pull request accepted, I might refactor some more ?

File: src/js/player.js [hunk 1]
```diff
@@ -1124,29 +1127,28 @@ class Player extends Component {
...
-          this.trigger('firstplay');
-        } else {
-          this.removeClass('vjs-has-started');
-        }
-      }
-      return;
+   *         the boolean value of hasStarted_
+   */
+  hasStarted(request) {
+    if (request === undefined || request === this.hasStarted_) {
```
Comment (gkatsev) [inline src/js/player.js]: A comment explaining that this is used as a getter or if the value isn't changing, we should just return what we have would be good

File: src/js/player.js [hunk 2]
```diff
@@ -1124,29 +1127,28 @@ class Player extends Component {
...
+    this.hasStarted_ = request;
+
+    if (this.hasStarted_) {
+      this.addClass('vjs-has-started');
+      this.trigger('firstplay');
+    } else {
+      this.removeClass('vjs-has-started');
+    }
+
+    return this.hasStarted_;
```
Comment (gkatsev) [inline src/js/player.js]: Our setters generally do not return anything.

Comment (misteroneill) [reply src/js/player.js]: As @gkatsev said, this is my only concern with the PR.

**Pull request merged**